content/90.back-matter.md

```diff
@@ -2,6 +2,25 @@
 
 The authors thank [Blair Sullivan](https://orcid.org/0000-0001-7720-6208) for [her feedback](https://github.com/greenelab/xswap-manuscript/issues/54) on a draft of the manuscript.
 
+## Availability of Supporting Source Code and Requirements
+
+Project name: XSwap
+
+Project homepage: https://github.com/greenelab/xswap-manuscript
+
+Operating system(s): MacOS, Linux, Windows
+
+Programming language: Python, C, C++
+
+Other requirements: None
+
+License: BSD 2-Clause
+
+RRID: SCR_024802
+
+biotoolsID: xswap
```

Comment:
Noting the following links just for convenience here, no edits needed:

https://scicrunch.org/resolver/RRID:SCR_024802
https://bio.tools/xswap